package/autoyast2.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Thu Mar 26 17:00:32 CET 2015 - schubi@suse.de
+
+- New autoinst flag in general/mode section: final_restart_services.
+  Restarting all services after finishing the installation.
+  The default is "true".
```

Comment:
I think you should explicitelly mention there that `The default is "true" which is backward compatible value.` so if you do not touch your autoyast profile it will be same as before.